website/data/cloud-docs-nav-data.json

```diff
@@ -616,34 +616,49 @@
   {
     "title": "Recommended Practices",
     "routes": [
-      { "title": "Overview", "path": "recommended-practices" },
       {
-        "title": "Part 1: Overview of Our Recommended Workflow",
-        "path": "recommended-practices/part1"
-      },
-      {
-        "title": "Part 2: Evaluating Your Current Provisioning Practices",
-        "path": "recommended-practices/part2"
-      },
-      {
-        "title": "Part 3: How to Evolve Your Provisioning Practices",
-        "path": "recommended-practices/part3"
-      },
-      {
-        "title": "Part 3.1: From Manual Changes to Semi-Automation",
-        "path": "recommended-practices/part3.1"
-      },
-      {
-        "title": "Part 3.2: From Semi-Automation to Infrastructure as Code",
-        "path": "recommended-practices/part3.2"
-      },
-      {
-        "title": "Part 3.3: From Infrastructure as Code to Collaborative Infrastructure as Code",
-        "path": "recommended-practices/part3.3"
+        "title": "Phases of Terraform Adoption",
```

Comment:
Talking more to Rita about this, she made a good point that we don't want to hide this content from Community users because it is a good part of the broader why-Cloud-is-good story. 

I'm going to talk to Adam about making the [Introduction to TF](https://developer.hashicorp.com/terraform/intro) section more discoverable, and we after we move it, I do think we should move this Phases of TF content to that Introduction section as you original proposed (sorry for going back and forth on this 😅 ). 

If you'd rather wait to add this to the Introduction section, I'd understand, but I still think this is a good in-between spot to get the content out now in the meantime.